src/euroeval/benchmark_modules/vllm.py

```diff
@@ -12,6 +12,7 @@
 from pathlib import Path
 from time import sleep
 
+import ray
```

Comment:
Can you wrap this in a `if t.TYPE_CHECKING or importlib.util.find_spec("ray") is not None` block, together with the corresponding vllm block below?